content/resources/operator/_index.md

```diff
@@ -81,7 +81,106 @@ Blood Pressure Reading:
 
-## 4. Programs and Projects
+
+## 4. Authentication Methods
+
+The following methods of authentication are supported in Gen3. They are listed in order of preference from OCC perspective, and technological and governance considerations are outlined below.
+
+### eRA Commons
+
+Authentication system developed by NIH for managing for research grants.
+
+_Pros_
+
+* Intended for officials, principal investigators, trainees and post-docs at institutions/organizations to access and share information relating to research grants.
+
+_Cons_
+
+* Can only be used if sponsored by an NIH institute.
+
+### Google (organizational account)
+
+Google Sign-In secure authentication system as provided by the user’s organization.
+
+_Pros_
+
+* User’s organization inherently takes responsibility for actions taken by user under the auspice of their organizational identity.
+* User’s organization takes responsibility for activating/deactivating and monitoring identity.
+
+_Cons_
+
+* Can only be used if the user’s organization utilizes Google for identity management.
+
+### Microsoft Office 365 (organizational account)
+
+Microsoft Office 365 secure authentication system as provided by the user’s organization.
+
+_Pros_
+
+* User’s organization inherently takes responsibility for actions taken by user under the auspice of their organizational identity.
+* User’s organization takes responsibility for activating/deactivating and monitoring identity.
+
+_Cons_
+
+* Can only be used if the user’s organization utilizes Microsoft Office 365 for identity management.
+
+### InCommon
+
+InCommon, operated by Internet2, provides a secure and privacy-preserving trust fabric for research and higher education, and their partners, in the United States. Individual identity providers, such as NIH iTrust and most academic institutions, are federated by InCommon.
+
+_Pros_
+
+* User’s organization inherently takes responsibility for actions taken by user under the auspice of their organizational identity.
+* User’s organization takes responsibility for activating/deactivating and monitoring identity.
+
+_Cons_
+
+* Can only be used if the user’s organization is part of the InCommon federation. 
+* Reference this resource for organizations supported by InCommon.
```

Comment:
Maybe I’m missing something, what is the con here?

Reply:
Not you, I'm missing links! Fixed in e59cb35